<?xml version="1.0"?>

<launch>
    <node name="em_tf2_srv" pkg="em_tf2" type="em_tf2_srv.py" output="screen" cwd="node" />
</launch>
